### PR TITLE
Consolidate Keep Pace feat into one

### PR DESCRIPTION
**Pull request by ammalagonc**
- Merged the two feats.
- ~~NOTICE - I did not add an Item Alteration to the description as the text output will not recognize the HTML formatting tags.~~
- updated the journal entries for the Bounty Hunter and Game Hunter to reference the merged feat.

### CODE REVIEW

**Comment by stwlam** [comment]
Markdown!

**Comment by ammalagonc** [comment]
Let's see if this works

File: static/lang/re-en.json [hunk 1]
```diff
@@ -3209,6 +3209,11 @@
                 }
             },
             "ItemStrike": "{item} Strike",
+            "KeepPace": {
+                "BountyHunter": "When your hunted prey tries to bolt, you follow. You Stride up to your Speed, following your hunted prey and keeping it in reach throughout its movement until it stops moving or you've moved your full Speed.<br> You can use Keep Pace to @UUID[Compendium.pf2e.actionspf2e.Item.H6v1VgowHaKHnVlG]{Burrow}, @UUID[Compendium.pf2e.actionspf2e.Item.pprgrYQ1QnIDGZiy]{Climb}, @UUID[Compendium.pf2e.actionspf2e.Item.cS9nfDRGD83bNU1p]{Fly}, or @UUID[Compendium.pf2e.actionspf2e.Item.c8TGiZ48ygoSPofx]{Swim} instead of Stride if you have the appropriate movement type.",
```
Comment (stwlam) [inline static/lang/re-en.json:3213]: I spy a `<br>`

Comment (ammalagonc) [reply static/lang/re-en.json]: Updated.

File: packs/feats/keep-pace.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        "prerequisites": {
+            "value": [
+                {
+                    "value": "Bounty Hunter Dedication"
+                },
+                {
+                    "value": "Game Hunter Dedication"
+                }
+            ]
+        },
```
Comment (TikaelSol) [inline packs/feats/keep-pace.json:28]:
```suggestion
        "prerequisites": {
            "value": [
                {
                    "value": "Bounty Hunter Dedication or Game Hunter Dedication"
                }
            ]
        },
```

**Pull request merged**